src/psyclone/domain/lfric/__init__.py

```diff
@@ -71,6 +71,7 @@
 from psyclone.domain.lfric.lfric_scalar_args import LFRicScalarArgs
 from psyclone.domain.lfric.lfric_loop_bounds import LFRicLoopBounds
 from psyclone.domain.lfric.lfric_kern_metadata import LFRicKernMetadata
+from psyclone.domain.lfric.lfric_dofmaps import LFRicDofmaps
```

Comment:
Not all your fault, but can you sort them alphabetically (unless there should be a dependency, then just ignore this).

Reply:
You're right, there are indeed dependencies which prevent us organizing these imports alphabetically.